pkgs/development/haskell-modules/generic-builder.nix

```diff
@@ -242,6 +242,9 @@ stdenv.mkDerivation ({
       if [ -d "$p/lib" ]; then
         configureFlags+=" --extra-lib-dirs=$p/lib"
       fi
+      if [[ -d "$p/Library/Frameworks" ]]; then
```

Comment:
So sometimes this is just in /Frameworks IIRC

Reply:
Then cc-wrapper needs to be changed too :D.

Reply:
Which is preferred? I think Library & System are often used by Apple kind if like usr is on Linux. At least it seems like we can move them to all be /Frameworks

Reply:
I have no idea. CC @NixOS/darwin-maintainers. I think it best we merge this and then sort out all that later.